sources/tools/Stride.Importer.FBX/Stride.Importer.FBX.vcxproj

```diff
@@ -223,6 +219,9 @@
       <Project>{5210fb81-b807-49bb-af0d-31fb6a83a572}</Project>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\engine\Stride.Rendering\Stride.Rendering.csproj">
+      <Project>{ad4fdc24-b64d-4ed7-91aa-62c9eda12fa4}</Project>
+    </ProjectReference>
```

Comment:
Not sure what's going on here, can you quickly go over why you had to remove the `<Private>False</Private>` here ?

Reply:
I had to manually update reference in Importer .vcxproj to the project Stride.Redenring vcproj. Each time I update the  Stride.Redenring vcproj, removing the reference and adding back in .vcxproj. It should be auto update but for some reason it was not. I will roll back so this goes to merge!